docs/week2/01-requirements.md

```
@@ -39,40 +45,45 @@
+
 - **결제 완료됨 (PAID)**: 결제가 완료되어 처리 대기 중인 상태
 
 **결제 상태:**
-- **준비 (READY)**: 결제가 준비된 상태
+
 - **완료 (PAID)**: 결제가 완료된 상태
 
 **행위:**
+
 - **주문하다 (place)**: 사용자가 상품을 주문하는 행위
 - **결제하다 (pay)**: 주문에 대한 결제를 완료하는 행위
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)